include/unixconf.h

```diff
@@ -139,7 +139,7 @@
 #define TIMED_DELAY
 #endif
 
-/* #define AVOID_WIN_IOCTL */ /* ensure USE_WIN_IOCTL remains undefined */
+#define AVOID_WIN_IOCTL /* ensure USE_WIN_IOCTL remains undefined */
```

Comment:
This comment could probably be added to, to help future clarity.
```suggestion
/* 
 * ensure USE_WIN_IOCTL remains undefined
 * USE_WIN_IOCTL can set LI and CO global variables for terminal size, which we fix in nle.c
 */
#define AVOID_WIN_IOCTL 
```

Reply:
formatting of the comment can obvs change.

Reply:
I see where you're coming from, but I'd like to stick with the policy of not unduly changing the original NetHack source files, and this is one of them.